### PR TITLE
updating locust to use Locust v0.9.0

### PR DESCRIPTION
**Pull request by sekka1**
#### What this PR does / why we need it:
This updates Locust to the latest v0.9.0.  It also uses the Dockerfile in the Locust Github repo as the standard image.  

#### Which issue this PR fixes
none

### CODE REVIEW

**Comment by sekka1** [comment]
@cpanato Here is the new PR with only the Locust v0.9.0 update.

**Comment by sekka1** [comment]
@cpanato can you approve?

**Comment by cpanato** [comment]
@sekka1 you need to sign the DCO

What happens with the previous image repo? is this new an official one?

**Comment by sekka1** [comment]
@unguiculus @davidkarlsen can you take a look at this for updating locust to the latest release?

**Comment by sekka1** [comment]
@cpanato dont seem like the maintainers of this chart is responding.  What would the next options be?

**Comment by so0k** [comment]
@sekka1 - what is the github repo for your Dockerfile?

the Honestbee Dockerfile is here - https://github.com/honestbee/distributed-load-testing/blob/master/locust-image/app/entrypoint.sh

the entrypoint for the workers used a loop to wait until the master became available, else you'd have a master that start without the workers registered. 

I don't know of Locust 0.9.0 has added fixes to avoid that scenario as I'm no longer at Honestbee and neither me or Honestbee is currently using Locust anymore.

If nobody of the Locust maintainers or community is willing to step up to manage these charts, we might just need to deprecate it - or add sources / references to the README so people can trust the docker images they run.

**Comment by sekka1** [comment]
@so0k Here is the Dockerfile:  https://github.com/locustio/locust/blob/master/Dockerfile

There dont seem to be any loops on the worker waiting for the master.  I didnt seem to run into this issue when i was using this.

Perhaps I should rather post this chart to the locust github repo instead and see if they will take it in?

**Comment by sekka1** [comment]
@so0k No answer to my question about them creating an offical docker image:  https://github.com/locustio/locust/issues/849

Can we move forward to merging this?

**Comment by quorak** [comment]
can we please support @sekka1 in getting this done. Current chart is very outdated and there seems to be a lot of interest!

File: stable/locust/templates/master-deploy.yaml [hunk 1]
```diff
@@ -10,6 +10,10 @@ metadata:
     component: master
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      component: master
+      app: {{ template "locust.fullname" . }}
```
Comment (cpanato) [inline stable/locust/templates/master-deploy.yaml:16]: please follow the doc: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#deployments-statefulsets-daemonsets-selectors

beside the `app` please add `release`

Comment (cpanato) [inline stable/locust/templates/master-deploy.yaml:16]:
```suggestion
      app: {{ template "locust.fullname" . }}
      release: {{ .Release.Name | quote }}
```

**Comment by blaketastic2** [comment]
Is there any update to getting this merged?

**Pull request closed without merging**